development/github-slack-connector/githubWebhookGateway/README.md

```diff
@@ -0,0 +1,47 @@
+# GitHub Webhook Gateway
+
+## Overview
+
+The Github Webhook Gateway is written in golang. It listens for incoming Github webhook events. Validates webhook hmac secret, converts it to CloudEvents compliant event and forwards it to the Kyma [Event Service](https://github.com/kyma-project/kyma/blob/master/components/event-service/README.md). It is meant to run within Kyma as a docker container.
+
+
+## Prerequisites
+
+Github Webhook Gateway was created to run within Kyma runtime, but it will work with any [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/spec.md) compliant receiver.
```

Comment:
```suggestion
Github Webhook Gateway was created to run within the Kyma Runtime, but it will work with any [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/spec.md) compliant receiver.
```